functions

```diff
@@ -1,10 +1,10 @@
-# DO NOT EDIT THIS FILE BY HAND -- YOUR CHANGES WILL BE OVERWRITTEN
 # Color code lookups
```

Comment:
is this not generated?  if it is you may want to put the banner in here, maybe the tool name and version at some point

Reply:
no shell opts set?

Reply:
It is generated, I meant to add that back in.  Correct, no shell opts is on purpose due to how this is used (via `BASH_ENV`).  It has been `shellcheck`'d.

Reply:
are you going to add it back?

Reply:
a7de832f58acde036e9a7c64b6a9aa01f23020ae